src/LineBot/ILineBot.cs

```diff
@@ -23,6 +23,13 @@ namespace Line
     /// </summary>
     public interface ILineBot
     {
+        /// <summary>
+        /// Creates a rich menu.
+        /// </summary>
+        /// <param name="richMenuRequest">The rich menu represented as a rich menu object.</param>
+        /// <returns>.</returns>
+        Task<string> CreateRichMenu(IRichMenuRequest richMenuRequest);
```

Comment:
Could you rename the type of the argument to `IRichMenu` instead so we match https://developers.line.me/en/reference/messaging-api/#rich-menu-object again? And again sorry for going back on this change,

Reply:
Surely, no worries.